src/apiProxy.js

```diff
@@ -51,7 +51,7 @@ export function getApiProxy(client, req, operationName) {
             }).catch((error) => {
               callInfo.error = error;
               logInfo.error = error;
-              log(req, client, 'error', 'elasticsearch failed', logInfo);
+              log(req, client, 'error', 'elasticsearch failed', { logInfo, args });
```

Comment:
Could this leak sensitive info? Seems like something that maybe should be configurable? Also I'd suggest logging a JSON.stringified version because logging flattens things in ways that could cause problems here.

Reply:
Generally when I stringify logs people tell me not to, and to use the object 3rd parameter instead. Since the log parser will break down the object better. This is my first time being told the inverse of that, but fair enough, will do.

Reply:
As far as I know we don't store any sensitive user info in elasticSearch, just station information. Also this will only log the queries not the results. So I don't think we'd ever have a _query_ that itself contained sensitive info... But I could be wrong.

Reply:
When you don't know what is in that hash (args), you need to encode. Because otherwise if that hash has "message", you will lose your message (or any other important key).

Reply:
(And to be clear the message string should almost never be interpolated)